pkg/driver/node/node.go

```diff
@@ -62,12 +63,13 @@ const (
 
 // S3NodeServer is the implementation of the csi.NodeServer interface
 type S3NodeServer struct {
-	NodeID  string
-	Mounter mounter.Mounter
+	NodeID    string
+	Mounter   mounter.Mounter
+	Clientset kubernetes.Interface
 }
 
-func NewS3NodeServer(nodeID string, mounter mounter.Mounter) *S3NodeServer {
-	return &S3NodeServer{NodeID: nodeID, Mounter: mounter}
+func NewS3NodeServer(nodeID string, mounter mounter.Mounter, clientset kubernetes.Interface) *S3NodeServer {
```

Comment:
I feel like it would be better if we don't change this file at all. Seems like we just store it in `S3NodeServer` to later access in `Driver`. Let's remove this one and store the `Clientset` in `Driver`.